controls/3.7/cf_serverd.cf

```diff
@@ -95,6 +95,15 @@ bundle server access_rules()
       comment => "Grant access to shell for cfruncommand",
       admit => { @(def.policy_servers) };
 
+    policy_server::
+      "/var/cfengine/def/phases.json"
+        handle => "server_access_grant_access_def_phases_json",
+        comment => "Grant access to phase definitions",
+        admit => { @(def.acl) };
+
+      "/var/cfengine/policychannel"
+        admit => { @(def.acl) };
+
```

Comment:
Acls to allow download of the phases data file, and allowing access to the root of the policy channels. Note, this could be considered a weak acl. It would be better if the acl for this was more precise and automaitcaly generated from the other data. But v1 ...